src/containers/Tenant/utils/newSQLQueryTemplates.ts

```diff
@@ -248,3 +248,11 @@ export const dropAsyncReplicationTemplate = () => {
 export const alterAsyncReplicationTemplate = () => {
     return `ALTER ASYNC REPLICATION \`$path\` SET (STATE=\`$state\`, FAILOVER_MODE=\`$failoverMode\`);`;
 };
+
+export const addTableIndex = () => {
```

Comment:
We may use function `addTableIndex('$path')` from `src/containers/Tenant/utils/schemaQueryTemplates.ts` . In most cases templates are identical.
What do you think if we create general template creators and use them if possible? This will prevent double-fixing templates.

Reply:
https://github.com/ydb-platform/ydb-embedded-ui/issues/1460

Reply:
on a second thougth this can be done right here